src/main.rs

```diff
@@ -470,6 +470,7 @@ fn run() -> Result<(), failure::Error> {
                 )
                 .arg(
                     Arg::with_name("release")
```

Comment:
if we deprecated it, should it just go away?

Reply:
eh - doesnt really hurt to keep it around so the command doesnt fail in case folks have it in CI but if you want we can remove

Reply:
if  it's going into a  point release, we should probably leave it. would be a shitty reason for your ci  to fail. maybe  it doesn't get pulled until 2.0? idk.

Reply:
yeah, makes sense